### PR TITLE
Dịch "mlp_vn" - Phần 1

### LINKED ISSUE #820: Dịch "mlp_vn" - Phần 1
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_multilayer-perceptrons/mlp_vn.md#L1-L68

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by di-mi-ta**
V1

## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #820

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 1]
```diff
@@ -18,20 +18,23 @@ Now that we have mastered these mechanics in the context of simple linear models
 we can launch our exploration of deep neural networks, the comparatively rich class of models with which this book is primarily concerned.
 -->
 
-*dịch đoạn phía trên*
+Trong chương trước, chúng tôi đã giới thiệu hồi quy softmax (:numref:`sec_softmax`), cách hiện thực giải thuật này từ đầu (:numref:`sec_softmax_scratch`) và trong gluon (:numref:`sec_softmax_gluon`), huấn luyện các bộ phân loại để nhận diện 10 lớp quần áo khác nhau từ các bức ảnh có độ phân giải thấp. 
+Cùng với đó, chúng ta đã học về cách sắp xếp dữ liệu, ép buộc các giá trị đầu ra tạo thành một phân phối xác suất hợp lệ (thông qua hàm `softmax`), áp dụng một hàm mất mát phù hợp và tối thiểu nó đối với các tham số của mô hình.
```
Comment (tiepvupsu) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
Cùng với đó, chúng ta đã học về cách sắp xếp dữ liệu, ép buộc các giá trị đầu ra tạo thành một phân phối xác suất hợp lệ (thông qua hàm `softmax`), áp dụng một hàm mất mát phù hợp và tối thiểu nó theo các tham số mô hình. 
```

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 2]
```diff
@@ -63,7 +67,11 @@ A increase in income from $0 to $50k likely corresponds to a bigger increase in
 One way to handle this might be to pre-process our data such that linearity becomes more plausible, say, by using the logarithm of income as our feature.
 -->
 
-*dịch đoạn phía trên*
+Ví dụ, tính tuyến tính ngụ ý trong đó một giả định *yếu hơn* của *tính đơn điệu*: nghĩa là bất kì một sự tăng lên nào của các đặc trưng luôn gây ra hoặc là sự tăng lên của đầu ra mô hình (nếu trọng số tương ứng dương), hoặc là sự giảm xuống của đầu ra mô hình (nếu trọng số tương ứng âm).
+Điều này đôi khi có ý nghĩa.
+Ví dụ, nếu chúng ta dự đoán liệu một cá nhân nào đó có khả năng trả được khoản vay hay không, chúng ta có thể suy diễn một cách hợp lý như sau, giữa các ứng viên có những điều kiện khác như nhau thì ứng viên nào có thu nhập cao hơn sẽ luôn có khả năng trả được nợ cao hơn so với những ứng viên khác có thu nhập thấp hơn. 
+Trong khi đó tính đơn điệu trong mối quan hệ này giống như việc tăng thu nhập từ $0 lên $50k tương ứng với khả năng trả được nợ lớn hơn so với mức tăng từ $1M lên $1.05M.
+Một cách để giải quyết điều này là tiền xử lý dữ liệu của chúng ta sao cho tính tuyến tính trở nên hợp lý hơn, bằng việc sử dụng logarit của thu nhập như là một đặc trưng.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
Một cách để giải quyết điều này là tiền xử lý dữ liệu của chúng ta để giả định tuyến tính trở nên hợp lý hơn, ví dụ như sử dụng logarit của thu nhập làm đặc trưng. 
```

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 3]
```diff
@@ -63,7 +67,11 @@ A increase in income from $0 to $50k likely corresponds to a bigger increase in
 One way to handle this might be to pre-process our data such that linearity becomes more plausible, say, by using the logarithm of income as our feature.
 -->
 
-*dịch đoạn phía trên*
+Ví dụ, tính tuyến tính ngụ ý trong đó một giả định *yếu hơn* của *tính đơn điệu*: nghĩa là bất kì một sự tăng lên nào của các đặc trưng luôn gây ra hoặc là sự tăng lên của đầu ra mô hình (nếu trọng số tương ứng dương), hoặc là sự giảm xuống của đầu ra mô hình (nếu trọng số tương ứng âm).
+Điều này đôi khi có ý nghĩa.
```
[the remark this answers is not in the document]
Comment (duythanhvn) [reply chapter_multilayer-perceptrons/mlp_vn.md]: Eh... mình thấy câu sửa giống văn nói hơn...

[the remark this answers is not in the document]
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/mlp_vn.md]: mình thấy bình thường mà :slightly_frowning_face: với lại `make sense` đâu phải `có ý nghĩa` đâu
how about this
```suggestion
Điều này đôi khi cũng hợp lý.
```

[the remark this answers is not in the document]
Comment (duythanhvn) [reply chapter_multilayer-perceptrons/mlp_vn.md]: Ổn hơn rồi ah.

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 4]
```diff
@@ -587,7 +595,7 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 
 * Đoàn Võ Duy Thanh
 <!-- Phần 1 -->
-*
+* Đinh Minh Tân
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
* Đinh Minh Tân
* Phạm Minh Đức
* Vũ Hữu Tiệp
```
